v3/src/components/case-card/case-card.scss

```diff
@@ -1,334 +1,366 @@
-@use "../vars.scss";
-
-$table-body-font-size: 8pt;
-
-.case-card {
-  position: relative;
+@import "../vars.scss";
+@import "./case-card-vars.scss";
+
+.codap-case-card {
+  overflow: auto hidden;
+}
+
+input.column-name-input,
+input.collection-title-input,
```

Comment:
These are actually case-card-specific styles. Sorry, I should've placed them all within the `.codap-case-card` rule set, which I've done in the latest commit. I placed them at this level because the input elements rendered by various Case Card child components share these styles.